src/Pages/ProductDetails/ProductDetails.js

```diff
@@ -0,0 +1,72 @@
+import { useParams } from "react-router-dom";
+import { useData } from "../../Contexts";
+import { Link } from "react-router-dom";
+import "./ProductDetails.css"
+
+export function ProductDetails() {
+    const { productId } = useParams();
+    const { data, itemsInCart, itemsInWishList, dispatch } = useData();
+    const product = data.find((item) => item.id === productId )
+    const {
+        id,
+        name,
+        image,
+        price,
+        material,
+        brand,
+        inStock,
+        fastDelivery,
+        ratings,
+        offer,
+        idealFor,
+        level,
+        color
+      } = product;
+
+    const CartButtons = () => {
+        if(itemsInCart.find((item) => item.id === id)){
+            return(
+                <Link to="/cart"><button class="btn btn-primary interactions-button-cart" >Go To Cart</button></Link>
+            )
+        }
+        return(
+            <button class="btn btn-primary interactions-button-cart" onClick={() => dispatch({type: "MANIPULATE_CART", payload: {manipulation: "ADD_TO_CART", item: product}})} >Add To Cart</button>
+        )
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/41542536/117572397-58008200-b0f0-11eb-8f91-e430fab8a578.png)
![image](https://user-images.githubusercontent.com/41542536/117572408-68186180-b0f0-11eb-841e-17fd15b7ddb6.png)

As you click on add to cart button the button changes to a link Go to cart and as you have used Link tag instead of a button the alignment changes.

Reply:
Fixed it, implemented navigate(), instead of Link